e2e/cypress/integration/files_and_attachments/file_previews_1_spec.js

```diff
@@ -14,7 +14,7 @@ import * as TIMEOUTS from '../../fixtures/timeouts';
 
 import {downloadAttachmentAndVerifyItsProperties} from './helpers';
 
-describe('Upload Files', () => {
+describe('Upload Files - Image', () => {
```

Comment:
I really like this grouping per file type, would it make sense to separate each file type into its own test case?  Feels like this single test case has a huge weight to be measured as a single unit of test.

Reply:
@saturninoabril I'm not quite sure I follow. Each file type is already it's own test case. You can see that each TM4J tag is assigned to each file type

Reply:
Oopss, sorry. I meant MM-T2264 for images, then new TM4J test case each for video, audio, generic, etc.

Reply:
Ah, gotcha.

Reply:
@saturninoabril updated